airflow/api_connexion/openapi/v1.yaml

```diff
@@ -872,7 +872,9 @@ paths:
         '403':
           $ref: '#/components/responses/PermissionDenied'
 
-  /dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/xcomEntries/{xcom_key}:
+  # TODO: /dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/xcomEntries/{xcom_key}:
```

Comment:
Can you add a description that will contain the necessary steps to take and the reason for the change?